### ISSUE TITLE
Support of keyboards

### ISSUE THREAD

**Issue by dev4openid** [egaga/openvim#120]
The application only supports US keyboards. For e.g. if using UK keyboards then many examples do not work as keyboard mappings are different.  
Perhaps consider a config where the country specific keyboard can be configured.

**Comment by unelma** [comment]
Yup. There is layout support for only dvorak currently https://openvim.com/?layout=dvorak
